File: Two Sum/twosum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        conjugate= {}
        for i in range(len(nums)):
            if str(target - nums[i]) in conjugate:
                temp = conjugate[str(target - nums[i])]
                firstindex = temp[1]
                return [firstindex, i]
            conjugate[str(nums[i])] = [target - nums[i], i]

File: Two Sum/twosum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        Dictionary<int,int> dict = new Dictionary<int,int>();
        int[] result = new int[2];
        for(int i = 0; i< nums.Length;i++)
        {
            var remainder = target - nums[i];
            if(dict.ContainsKey(remainder))
            {
                return new int[]{i, dict[remainder]};
            }
            if(!dict.ContainsKey(nums[i]))
            {
                dict.Add( nums[i], i);
            }
        }
        return result;
    }
}